typeA/typeA.ts

```diff
@@ -0,0 +1,92 @@
+export interface typeANotice {
```

Comment:
```suggestion
export interface TypeANotice {
```
보통 타입에는 PascalCase (첫글자가 대문자인 camelCase) 를 사용해요.